### PR TITLE
[ETCM-403] Expose application.conf and chains configs directory

### PR DESCRIPTION
**Pull request by mmrozek**
# Description

Copy all configs to the package conf directory - it will be easier to understand for users how to configure the client.

### CODE REVIEW

File: src/universal/conf/app.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # This is the base configuration file for the Mantis ETC client.
```
Comment (mirkoAlic) [inline src/universal/conf/app.conf:1]: maybe is a good time to get rid of this file(`app.conf`). IMO is just a useless proxy.

Comment (mirkoAlic) [reply src/universal/conf/app.conf:1]: (disclaimer: probably out of scope)

**Review by mirkoAlic — APPROVED** [review]
LGTM!

**Review by faraggi — COMMENTED** [review]
LGTM :)

**Pull request merged**